src/main/java/entity/Collectible.java

```diff
@@ -0,0 +1,32 @@
+package entity;
+
+import entity.Item;
+
+public class Collectible extends Item {
+
+    /**
+     * Creates a new entity.Collectible object.
+     * @param collectibleType the type of this entity.Collectible.
+     */
+    public Collectible(String collectibleType){
+        super(collectibleType);
+    }
+
+    /**
+     * Creates a new entity.Collectible object.
+     * @param collectibleType the type of this entity.Collectible.
+     * @param num the amount of this entity.Collectible.
+     */
+    public Collectible(String collectibleType, int num){
+        super(collectibleType, num);
+    }
+
+    /**
+     * Sets the num of the entity.Collectible to the collectibleNum given.
+     * @param collectibleNum the new value for num.
+     */
+    public void setCollectibleNum(int collectibleNum){
```

Comment:
Do we need to check if there are enough collectibleNum to decrease?

Reply:
I had the checkMinusAvailable() method before in the Item class, but I removed it because I wasn't sure if it was necessary. This is because you can just use the getNum() method. But if you think it would be helpful I'll add checkMinusAvailable().